server/rest/src/test/java/org/infinispan/rest/AuthenticationTest.java

```diff
@@ -76,7 +76,7 @@ public void shouldRejectNotValidAuthorizationString() throws Exception {
 
       BasicAuthenticator basicAuthenticator = new BasicAuthenticator(securityDomainMock, "ApplicationRealm");
 
-      restServer = RestServerHelper.defaultRestServer().withAuthenticator(basicAuthenticator).start();
+      restServer = RestServerHelper.defaultRestServer().withAuthenticator(basicAuthenticator).start(this.getTestName());
```

Comment:
How about getting current's thread name (which would be probably a Surefire fork) rather than grabbing test name explicitly. This way the start method wouldn't need to accept parameters.

Reply:
Well, this is a test helper, so I'm not bothered about parameters. I've aligned the naming to use the AbstractInfinispanTest strategy.

Reply:
+1 to keep the parameter. 

However, `getTestName()` returns the full class name, so I suggest using `TestResourceTracker.getCurrentTestShortName()` instead.